CHANGES.md

```diff
@@ -57,7 +57,7 @@
 
 ## Highlights
 
-* Added FlinkRunner for Flink 1.17, removed support for Flink 1.12 and 1.13. Previous version of Pipeline running on Flink 1.16 and below can be upgraded to 1.17, if the Pipeline is first updated to Beam 2.56.0 with the same Flink version. After Pipeline runs with Beam 2.56.0, it should be possible to upgrade to FlinkRunner with Flink 1.17. ([#29939](https://github.com/apache/beam/issues/29939))
+* Added FlinkRunner for Flink 1.17 and 1.18, removed support for Flink 1.12 and 1.13. Previous version of Pipeline running on Flink 1.16 and below can be upgraded to 1.17 or 1.18, if the Pipeline is first updated to Beam 2.56.0 with the same Flink version. After Pipeline runs with Beam 2.56.0, it should be possible to upgrade to FlinkRunner with Flink 1.17 or 1.18. ([#29939](https://github.com/apache/beam/issues/29939))
```

Comment:
Beam 2.56.0 is already cut. 1.18 support will go to 2.57.0. Could add a new section in CHANGES.md copy-paste from the template above

Reply:
Sure thing!